### PR TITLE
Enhancement/#101 explanatory popup next to coatrack service identifier

### PR DESCRIPTION
**Pull request by ChristophBaierATB**
closes coatrack#101

There is a new info box label in the 'Edit Service' menu which is next to 'CoatRack Service Identifier'.

### CODE REVIEW

File: spring-boot/admin/src/main/resources/messages.properties [hunk 1]
```diff
@@ -178,7 +178,9 @@ admin.help.getting.started.service.url.text= Please insert the URL to which the
 admin.help.getting.started.service.url.image.path=/images/coatrack-getting-started-service-url.png
 admin.help.getting.started.service.url.image.alt=CoatRack overview and first steps - Service URL
 admin.help.getting.started.service.identifier.heading=Unique identifier for your service
-admin.help.getting.started.service.identifier.text= Service users need to use this identifier as part of the URL so that the CoatRack gateway will know to which service this call should be forwarded.
+admin.help.getting.started.service.identifier.warning.heading=Service Identifier editing warning
+admin.help.getting.started.service.identifier.explanatory.text= Service users need to use this identifier as part of the URL so that the CoatRack gateway will know to which service this call should be forwarded.
+admin.help.getting.started.service.identifier.caution.text=Be careful about editing the service identifier! It will change the URL that is used to access the service via the CoatRack gateway which will make existing API keys unusable.
```
Comment (gr-hovest-atb) [inline spring-boot/admin/src/main/resources/messages.properties:183]: The following part of the text is not correct, please remove it:
`[...] which will make existing API keys unusable.`

As there is a `.warning.heading`, maybe use `.warning.text` as key for this text?

File: spring-boot/admin/src/main/resources/templates/admin/fragments/services/edit.html [hunk 2]
```diff
@@ -51,6 +51,12 @@ <h2 th:text="#{admin.services.edit.form.title}">
                                 <div class="form-group">
                                     <label class="control-label col-md-3 col-sm-3 col-xs-12">
                                         <span th:text="#{admin.services.edit.form.label.id}"/>
+                                        <a data-toggle="modal" data-target=".getting-started-service-identifier-modal">
+                                            <i class="fa fa-question-circle"></i>
+                                        </a>
```
Comment (gr-hovest-atb) [inline spring-boot/admin/src/main/resources/templates/admin/fragments/services/edit.html:56]: To make the message even more obvious, I would suggest to add:
```suggestion
                                        </a>
                                        <br/>
                                        <small style="color: red">Be careful when editing</small>
```

File: spring-boot/admin/src/main/resources/messages.properties [hunk 3]
```diff
@@ -178,7 +178,9 @@ admin.help.getting.started.service.url.text= Please insert the URL to which the
 admin.help.getting.started.service.url.image.path=/images/coatrack-getting-started-service-url.png
 admin.help.getting.started.service.url.image.alt=CoatRack overview and first steps - Service URL
 admin.help.getting.started.service.identifier.heading=Unique identifier for your service
-admin.help.getting.started.service.identifier.text= Service users need to use this identifier as part of the URL so that the CoatRack gateway will know to which service this call should be forwarded.
+admin.help.getting.started.service.identifier.warning.heading=Service Identifier editing warning
```
Comment (gr-hovest-atb) [inline spring-boot/admin/src/main/resources/messages.properties:181]:
```suggestion
admin.help.getting.started.service.identifier.warning.heading=Be careful when editing the Service Identifier!
```

**Review by gr-hovest-atb — APPROVED** [review]
We should maybe change the color of the warning message, as the current "red" looks like an error. Leaving that aside, this looks good to me, I would suggest a PR to coatrack master.

**Comment by ChristophBaierATB** [comment]
This PR is going to be merged to the CoatRack main repository.

**Pull request closed without merging**